containers/src/Data/IntMap/Internal.hs

```diff
@@ -598,7 +599,23 @@ notMember k m = not $ member k m
 lookup :: Key -> IntMap a -> Maybe a
 lookup !k = go
   where
-    go (Bin p m l r) | nomatch k p m = Nothing
+    go (Bin _p m l r)
+                     | zero k m  = go l
+                     | otherwise = go r
```

Comment:
```suggestion
    go (Bin _p m l r)  | zero k m  = go l
                       | otherwise = go r
```
This doesn't really matter, but it's consistent with the rest of the definition.